# bboxes 
# [x1,y1,x2,y2,idx,class]
Track_6[] tracks 